src/pages/Dataslate.tsx

```diff
@@ -104,6 +114,15 @@ const Dataslate = () => {
           {hash === '#background' && <TeamBackground />}
         </section>
       </div>
+      <ConfirmModal
+        showModal={showDeleteDataslateModal}
+        message={'Are you sure you want to delete Dataslate'}
+        onConfirm={() => {
+          console.log('log: ' + deleteDataslate(dataslate))
```

Comment:
Remove the call from a log.

Add an action to the store to delete the dataslate that then calls the new function. 
It seems silly but we want the components to only use the stores.

If you would regather we can also merge this and make a ticket to add it to the store instead.